src/test/regress/expected/foreign_key_restriction_enforcement_0.out

```diff
@@ -1064,8 +1064,6 @@ ADD CONSTRAINT
 	fkey_delete FOREIGN KEY(value_1) 
 REFERENCES 
 	reference_table(id) ON DELETE CASCADE;
-DEBUG:  switching to sequential query execution mode
-DETAIL:  Reference relation "reference_table" is modified, which might lead to data inconsistencies or distributed deadlocks via parallel accesses to hash distributed relations due to foreign keys. Any parallel modification to those hash distributed relations in the same transaction can only be executed in sequential query execution mode
```

Comment:
Can we add a `SHOW citus.multi_shard_modify_mode;` here to check that connection is sequential?

Reply:
We're not inside a transaction block, and as far as I understand this test aims to make sure that adding fkey to reference table works without in a tx block.

If we cannot switch to sequential mode, we'd simply get into a self distributed-deadlock. So, is that good enough for you?